tools/dds/dds-server/rs-dds-server.cpp

```diff
@@ -96,114 +96,172 @@ void add_init_device_header_msg( rs2::device dev, std::shared_ptr<librealsense::
 {
     using namespace librealsense::dds::topics;
    
-    device::notifications::device_header device_header_msg;
+    device::notification::device_header_msg device_header_msg;
     auto &&sensors = dev.query_sensors();
-    device_header_msg.num_of_streams = sensors.size();
+    device_header_msg.num_of_sensors = sensors.size();
 
-    raw::device::notifications raw_msg;
-    device::notifications::construct_raw_message(
-        device::notifications::msg_type::DEVICE_HEADER,
+    raw::device::notification raw_msg;
+    device::notification::construct_raw_message(
+        device::notification::msg_type::DEVICE_HEADER,
         device_header_msg,
         raw_msg );
 
     server->add_init_msgs( raw_msg );
 }
 
-void send_stream_header_msg(  std::shared_ptr<librealsense::dds::dds_device_server> &server, rs2::sensor sensor, const int stream_idx, size_t profiles_count )
+void send_sensor_header_msg(  std::shared_ptr<librealsense::dds::dds_device_server> &server, rs2::sensor sensor, librealsense::dds::topics::device::notification::sensor_header_msg::sensor_type sensor_type, const int sensor_idx )
```

Comment:
`server` won't be changed, right? Then the reference should be `const`

Reply:
👍